lib/l10n/app_en.arb

```diff
@@ -676,10 +702,16 @@
     "l_rfc4514_invalid": "Invalid RFC 4514 format",
     "rfc4514_examples": "Examples:\nCN=Example Name\nCN=jsmith,DC=example,DC=net",
     "s_allow_fingerprint": "Allow fingerprint",
+    "s_pin_policy": "PIN policy",
+    "s_output_format": "Output format",
+    "s_expiration_date": "Expiration date",
+    "p_algorithm_desc": "Key algorithm to use.",
+    "p_output_format_desc": "The public key can be:\n\t• Saved to file.\n\t• Embedded into a self-signed certificate stored on the YubiKey (default).\n\t• Saved to file in a certificate signing request (CSR).",
```

Comment:
I wasn't able to find this string in the app, is it used?

Reply:
It has now been removed.